packages/in_app_purchase/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## [0.0.1]
```

Comment:
We're not using the square brackets in other plugins. Do we need to change the template?